### PR TITLE
Delete publishing validation in release 6.0 

### PR DESCRIPTION
**Pull request by epananth**
Delete publishing validation stage in release 6.0, because we have use always use main for build  promotion

### CODE REVIEW

**Comment by epananth** [comment]
https://dev.azure.com/dnceng/internal/_build/results?buildId=2411244&view=results

**Review by riarenas — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -195,37 +195,10 @@ extends:
...
-                  -azdoOrg "dnceng"
-                  -azdoProject "internal"
-                  -barToken $(MaestroAccessToken)
-                  -githubPAT $(BotAccount-dotnet-bot-repo-PAT)
       - stage: Promote_to_net6_eng_channel
         displayName: Promote Arcade to '.NET 6 Eng' channel
-        dependsOn:
-         - Validate_Publishing
+        dependsOn: 
+          - publish_using_darc
```
Comment (riarenas) [inline azure-pipelines.yml:201]: Does the indentation change here affect anything?

Comment (epananth) [reply azure-pipelines.yml:201]: Looks like the change worked https://dev.azure.com/dnceng/internal/_build/results?buildId=2411244&view=logs&s=0d7db6e4-c533-50c8-77c7-b1b9fc049ed6

**Comment by epananth** [comment]
@mmitche can you please merge this one, I don't have access to merge

**Pull request merged**